custom-commands/silverstripe/sake

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+vendor/bin/sake $@
```

Comment:
Sorry, I should have said this before, but `vendor/bin` is already in the $PATH, so you could just have `sake` here.

Reply:
Oh, cool! That indeed works just fine! :+1: